### PR TITLE
Feature/filters relationship

### PR DESCRIPTION
**Pull request by eliasassuncao**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# Unreleased
+
+### Added
+- added option to pass relationship in filters.
```
Comment (DouglasCalora) [inline CHANGELOG.md:10]: to send

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
...
+- added `preQueryList` function to intercept queries.
+- added the possibility to have relationship of collections only for declared methods.
```
Comment (DouglasCalora) [inline CHANGELOG.md:12]: possibility of having

**Review by DouglasCalora — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (eliasassuncao) [reply CHANGELOG.md:10]: feito.

File: CHANGELOG.md (hunk 2 above)
Comment (eliasassuncao) [reply CHANGELOG.md:12]: feito.

**Review by caiotarifa — APPROVED** [review]

**Review by DouglasCalora — APPROVED** [review]

**Pull request merged**